ui/DesignSystem/Component/TrackToggle.svelte

```diff
@@ -1,6 +1,8 @@
-<script lang="ts">
+<script lang="typescript">
```

Comment:
```suggestion
<script lang="ts">
```

Unless there's a good reason for the change, let's keep it consistent.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1030#discussion_r504479002

Reply:
Ah cool :+1: